static-assets/js/tinymce-plugins/ace/plugin.min.js

```diff
@@ -1 +1 @@
>]*>.*?<\/script(?=[>\s])|\/?(\?xml(:\w+)?|img|meta|link|style|\w:\w+)(?=[\s\/>]))[^>]*>/gi,[/<(\/?)s>/gi,"<$1strike>"],[/&nbsp;/gi," "],[/<span\s+style\s*=\s*"\s*mso-spacerun\s*:\s*yes\s*;?\s*"\s*>([\s\u00a0]*)<\/span>/gi,(t,e)=>e.length>0?e.replace(/./," ").slice(Math.floor(e.length/2)).split("").join(" "):""]];s=t.filter(s,o);const r=new tinymce.html.DomParser({},e.schema);r.addAttributeFilter("style",t=>{let s,o=t.length;for(;o--;)(s=t[o]).attr("style",n(e,s,s.attr("style"))),"span"===s.name&&s.parent&&!s.attributes.length&&s.unwrap()}),r.addAttributeFilter("class",t=>{let e,n,s=t.length;for(;s--;)n=(e=t[s]).attr("class"),/^(MsoCommentReference|MsoCommentText|msoDel)$/i.test(n)&&e.remove(),e.attr("class",null)}),r.addNodeFilter("del",t=>{let e=t.length;for(;e--;)t[e].remove()}),r.addNodeFilter("a",t=>{let e,n,s,o=t.length;for(;o--;)if(n=(e=t[o]).attr("href"),s=e.attr("name"),n&&-1!==n.indexOf("#_msocom_"))e.remove();else if(n&&0===n.indexOf("file://")&&(n=n.split("#")[1])&&(n="#"+n),n||s){if(s&&!/^_?(?:toc|edn|ftn)/i.test(s)){e.unwrap();continue}e.attr({href:n,name:s})}else e.unwrap()});const l=r.parse(s);return s=(new tinymce.html.Serializer).serialize(l)})(s,o)),o=((t,e)=>{const n=t.settings.paste_word_valid_elements||"-strong/b,-em/i,-u,-span,-p,-ol,-ul,-li,-h1,-h2,-h3,-h4,-h5,-h6,-h7,-h8,-pre,-p/div,-a[href|name],sub,sup,pre,blockquote,strike,br,del,table[width],tr,td[colspan|rowspan|width],th[colspan|rowspan|width],thead,tfoot,tbody",s=new tinymce.html.Schema({valid_elements:n,valid_children:"-li[p]"}),o=new tinymce.html.DomParser({},s);o.addAttributeFilter("style",t=>{let e,n=t.length;for(;n--;)(e=t[n]).attr("style",null)}),o.addAttributeFilter("class",t=>{let e,n=t.length;for(;n--;)(e=t[n]).attr("class",null)}),o.addNodeFilter("p",t=>{let e,n=t.length;for(;n--;){let s=(e=t[n]).firstChild,o=!0;for(;s;){if("br"!==s.name.toLowerCase()){o=!1;break}s=s.next}o&&e.unwrap()}});const r=o.parse(e);return e=(new tinymce.html.Serializer).serialize(r)})(s,o),s.execCommand("mceInsertContent",!1,o)};tinymce.PluginManager.add("paste_from_word",function(t,e){let n=!1;t.ui.registry.addToggleButton("paste_as_text",{active:!1,icon:"paste-text",tooltip:"Paste as text",onAction:function(e){n=!n,e.setActive(n),t.focus()},onSetup:function(e){t.on("PastePlainTextToggle",function(n){e.setActive(n.state),t.focus()})}}),t.ui.registry.addToggleMenuItem("paste_as_text",{text:"Paste as text",icon:"paste-text",tooltip:"Paste as text",onAction:function(e){n=!n,t.fire("PastePlainTextToggle",{state:n})},onSetup:function(t){return t.setActive(n),function(){}}}),t.on("paste",function(e){e.preventDefault();const r=((t,e)=>(t=>{const e={};if(t){if(t.getData){const n=t.getData("Text");n&&n.length>0&&-1===n.indexOf("data:text/mce-internal,")&&(e["text/plain"]=n)}if(t.types)for(let n=0;n<t.types.length;n++){const s=t.types[n];try{e[s]=t.getData(s)}catch(t){e[s]=""}}}return e})(e.clipboardData||t.getDoc().dataTransfer))(t,e),l=void 0!==r["text/html"];!n&&l?o(r,t):s(r,t)})})}();
```

Comment:
The PR is missing the changes in the source/unminified version (i.e. `static-assets/js/tinymce-plugins/ace/plugin.js`) from which we build this minified file.

Reply:
My bad. I reverted `ace/plugin.js`